src/middleware.js

```diff
@@ -112,6 +114,62 @@ function errorHandler(err, req, res, next) {
   }
 }
 
+function cache(res) {
```

Comment:
I think `cache` should be an object vs a function, so you'd use it like this:

```js
app.get('/', cache.forever(), (res, res) => {...});
```

That means:

```js
const cache = {
  forever() {
    const t = ms('1y');

    return function(req, res, next) {
      res.set('Cache-Control', `public, max-age=${t}, immutable`);
      next();
    };
  },

  never(req, res, next) {
    ...
  }, 

  ...
};
```

Also, note the signature `(req, res, next)` that all Express middleware needs, and you have to call `next()` when done.  Last, `res.set()` vs. `res.headers.set()`.